api/core.py

```diff
@@ -416,7 +430,11 @@ def _archivestream(self, ticket):
         stream = cStringIO.StringIO()
         with tarfile.open(mode='w|', fileobj=stream) as archive:
             for filepath, arcpath, _ in ticket['target']:
-                yield archive.gettarinfo(filepath, arcpath).tobuf()
+                try:
+                    yield archive.gettarinfo(filepath, arcpath).tobuf()
+                except Exception as e:
```

Comment:
@rentzso has chosen a pragmatic solution for now. Imagine a scenario, where a user encounters a missing file after having received data for many hours and instead of letting them keep what they have, we throw it all away. That would surely piss me off. Maybe placing a README file into the tarball that lists any oddities about the download could be a good compromise.